### ISSUE TITLE
View remote telescope position, controlled by other computer

### ISSUE THREAD

**Issue by maxcaimmi** [Stellarium/stellarium#1181]
![screen1](https://user-images.githubusercontent.com/68903710/88656687-ffb44280-d0d0-11ea-98c6-c2cf8b68196b.jpg)
I have a telescope controlled by a remote computer in a home network (let's call it OAV-01) with Win XP3 + EQMOD Ascom (Skywatcher EQ6 pro mount) that works automatically through scripts.
I would need to view the position of the remote telescope, on Stellarium v0.20.0 installed on another computer of the same home network (let's call it OAV-00) with Windows 10 home.
I tried several times with the "Telescope Control" Plugin but it remains fixed in "connecting" ... is it possible to do what I need?
If yes, can you help me in the exact configuration for the above telescope setup?
Thank you!

**Comment by alex-w** [comment]
Are you read Help tab in the plugin?

**Comment by maxcaimmi** [comment]
Yes, not in great detail, but yes why?  Be nice, be clearer... what are you referring to?  (maybe I got the wrong forum, I'm not a computer engineer)-------- Messaggio originale --------Da: Alexander Wolf <notifications@github.com>Data: mar 28 lug 2020, 13:26A: Stellarium/stellarium <stellarium@noreply.github.com>Cc: maxcaimmi <info@oav.name>, Author <author@noreply.github.com>Oggetto: Re: [Stellarium/stellarium] View remote telescope position, controlled by other computer (#1181)
Are you read Help tab in the plugin?

**Comment by alex-w** [comment]
>Yes, not in great detail, but yes why?  Be nice, be clearer... what are you referring to?  (maybe I got the wrong forum, I'm not a computer engineer)

Your questions said me that you are not readed the help in the plugin (nor Stellarium User Guide).

First line of "Abilities and limitations" in help:
> This plug-in allows Stellarium to send only 'slew' ('go to') commands to the device and to receive its current position. It cannot issue any other commands, so users should be aware of the possibility for mount collisions and similar situations. (To abort a slew, you can start another one to a safe position.)

For indirect connection you may read:
> A device is connected to the same computer but it is driven by a stand-alone telescope server program or a third-party application that can 'talk' to Stellarium

So. TCP cannot show remote telescope position, controlled by other computer.

**Comment by gzotti** [comment]
I think the critical limitation is that only one computer can connect to a telescope at the same time. Either control the telescope with some other program, or with Stellarium, but not both.

**Comment by gzotti** [comment]
I just saw mention of an "ASCOM hub" which could help doing just that. I know zero details though, please search for that yourself.

**Comment by rkinnett** [comment]
> So, TCP cannot show remote telescope position, controlled by other computer.
It can if there is a proxy program on the remote computer, hosting both a direct connection to the mount and a TCP connection to Stellarium.  ASCOM Hub does not host a TCP socket with Stellarium telescope protocol and can't serve as a remote link between Stellarium and EQASCOM as the OP requested.  I think that functionality is out of scope of Stellarium.  It has to be implemented as a separate application.  I can make a python-based proxy if anyone is interested.  Perhaps that, as a proof of concept, can justify closing this ticket without (out-of-scope) changes to Stellarium?
